### ISSUE TITLE
[DOCS] Document known issue: Rule import fails if exceptions contain comments

### ISSUE THREAD

**Issue by peluja1012** [elastic/security-docs#1557]
## Description 

We should add a Known Issue for the following [bug](https://github.com/elastic/kibana/issues/124742). In 8.0, users have the ability to import rules along with their exceptions. However, if these exceptions contain comments, the import process will fail. The import process will work as expected if not comments are present in the exceptions being imported.

**Comment by yctercero** [comment]
### Workaround

The bug exists because when creating the exception list during import it fails validations for the `exceptionItem.comments`. We don't trust meta information from the client like `created_at`, `created_by`, `updated_at`, `updated_by`, these are system generated values and our validations reflect that.

Comments upon export might look something like:
```json
{
    "comment": 'this exception item rocks',
     "created_at": '2022-01-10T22:08:26.466Z',
     "created_by": 'user1',
     "id": '63caf778-3e54-4a23-a77a-ca14e3323fe2'
}
```

To bypass the bug, a user will need to remove fields so all that's left is:
```json
{
    "comment": 'this exception item rocks',
}
```

The exception item with its comments will be created. Just please note that it gets created with the information of the person who imported. So if "BugsBunny" imported the above existing comment, they'll see the created comment look something like:
```json
{
    "comment": 'this exception item rocks',
     "created_at": '2022-02-10T22:08:26.466Z',
     "created_by": 'BugsBunny',
     "id": '1234'
}
```

**Comment by nastasha-solomon** [comment]
@yctercero thanks for providing these notes! How are we recommending that users remove fields that the client adds and what steps should they take after removing the fields?

**Comment by yctercero** [comment]
NOTE: Followed up with @nastasha-solomon offline!

Natasha, here are the examples we chatted about. 

### Example export
```ndjson
{"id":"f6b2e330-89d9-11ec-9a39-157f181d84ab","updated_at":"2022-02-09T18:56:18.998Z","updated_by":"elastic","created_at":"2022-02-09T18:56:18.998Z","created_by":"elastic","name":"Rule w exceptions","tags":[],"interval":"30s","enabled":true,"description":"Sample rule with exception list","risk_score":1,"severity":"high","output_index":".siem-signals-default","author":[],"false_positives":[],"from":"now-6m","rule_id":"query-with-exceptions","max_signals":100,"risk_score_mapping":[],"severity_mapping":[],"threat":[],"to":"now","references":[],"version":1,"exceptions_list":[{"id":"55acf6b0-89d9-11ec-9a39-157f181d84ab","list_id":"simple_list","namespace_type":"single","type":"detection"}],"immutable":false,"type":"query","language":"kuery","query":"host.name: *","throttle":"no_actions","actions":[]}
{"_version":"WzI5ODUsMV0=","created_at":"2022-02-09T18:51:47.995Z","created_by":"elastic","description":"This is a sample detection type exception","id":"55acf6b0-89d9-11ec-9a39-157f181d84ab","immutable":false,"list_id":"simple_list","name":"Sample Detection Exception List","namespace_type":"single","os_types":[],"tags":["user added string for a tag","malware"],"tie_breaker_id":"3fadc899-230d-4f1e-97b4-65fd57f5469a","type":"detection","updated_at":"2022-02-09T18:51:47.998Z","updated_by":"elastic","version":1}
{"_version":"WzEzNzQwLDFd","comments":[{"created_at":"2022-02-10T23:29:56.447Z","comment":"I'm a comment","id":"5cfe008e-f88d-4556-98c4-e95dc1f4370b","created_by":"elastic"}],"created_at":"2022-02-09T20:11:39.280Z","created_by":"elastic","description":"Rule w exceptions - exception list item","entries":[{"field":"_index","type":"match","value":",nkjh","operator":"included"}],"id":"7d813010-89e4-11ec-9a39-157f181d84ab","item_id":"a09d05c7-495c-40ff-b8a8-3ceafd184951","list_id":"simple_list","name":"Rule w exceptions - exception list item","namespace_type":"single","os_types":[],"tags":[],"tie_breaker_id":"014c8b06-1ce0-4f6d-a97c-88795f27e760","type":"simple","updated_at":"2022-02-10T23:29:56.454Z","updated_by":"elastic"}
{"exported_count":3,"exported_rules_count":1,"missing_rules":[],"missing_rules_count":0,"exported_exception_list_count":1,"exported_exception_list_item_count":1,"missing_exception_list_item_count":0,"missing_exception_list_items":[],"missing_exception_lists":[],"missing_exception_lists_count":0}
```

### Example of import file after adjusting with workaround
```ndjson
{"id":"f6b2e330-89d9-11ec-9a39-157f181d84ab","updated_at":"2022-02-09T18:56:18.998Z","updated_by":"elastic","created_at":"2022-02-09T18:56:18.998Z","created_by":"elastic","name":"Rule w exceptions","tags":[],"interval":"30s","enabled":true,"description":"Sample rule with exception list","risk_score":1,"severity":"high","output_index":".siem-signals-default","author":[],"false_positives":[],"from":"now-6m","rule_id":"query-with-exceptions","max_signals":100,"risk_score_mapping":[],"severity_mapping":[],"threat":[],"to":"now","references":[],"version":1,"exceptions_list":[{"id":"55acf6b0-89d9-11ec-9a39-157f181d84ab","list_id":"simple_list","namespace_type":"single","type":"detection"}],"immutable":false,"type":"query","language":"kuery","query":"host.name: *","throttle":"no_actions","actions":[]}
{"_version":"WzI5ODUsMV0=","created_at":"2022-02-09T18:51:47.995Z","created_by":"elastic","description":"This is a sample detection type exception","id":"55acf6b0-89d9-11ec-9a39-157f181d84ab","immutable":false,"list_id":"simple_list","name":"Sample Detection Exception List","namespace_type":"single","os_types":[],"tags":["user added string for a tag","malware"],"tie_breaker_id":"3fadc899-230d-4f1e-97b4-65fd57f5469a","type":"detection","updated_at":"2022-02-09T18:51:47.998Z","updated_by":"elastic","version":1}
{"_version":"WzEzNzQwLDFd","comments":[{"comment":"I'm a comment"}],"created_at":"2022-02-09T20:11:39.280Z","created_by":"elastic","description":"Rule w exceptions - exception list item","entries":[{"field":"_index","type":"match","value":",nkjh","operator":"included"}],"id":"7d813010-89e4-11ec-9a39-157f181d84ab","item_id":"a09d05c7-495c-40ff-b8a8-3ceafd184951","list_id":"simple_list","name":"Rule w exceptions - exception list item","namespace_type":"single","os_types":[],"tags":[],"tie_breaker_id":"014c8b06-1ce0-4f6d-a97c-88795f27e760","type":"simple","updated_at":"2022-02-10T23:29:56.454Z","updated_by":"elastic"}
{"exported_count":3,"exported_rules_count":1,"missing_rules":[],"missing_rules_count":0,"exported_exception_list_count":1,"exported_exception_list_item_count":1,"missing_exception_list_item_count":0,"missing_exception_list_items":[],"missing_exception_lists":[],"missing_exception_lists_count":0}
```

At a more granular level, here's the sample of just the relevant exception item.

### Exception item as shown in export/import file
```ndjson
{"_version":"WzEzNzQwLDFd","comments":[{"created_at":"2022-02-10T23:29:56.447Z","comment":"I'm a comment","id":"5cfe008e-f88d-4556-98c4-e95dc1f4370b","created_by":"elastic"}],"created_at":"2022-02-09T20:11:39.280Z","created_by":"elastic","description":"Rule w exceptions - exception list item","entries":[{"field":"_index","type":"match","value":",nkjh","operator":"included"}],"id":"7d813010-89e4-11ec-9a39-157f181d84ab","item_id":"a09d05c7-495c-40ff-b8a8-3ceafd184951","list_id":"simple_list","name":"Rule w exceptions - exception list item","namespace_type":"single","os_types":[],"tags":[],"tie_breaker_id":"014c8b06-1ce0-4f6d-a97c-88795f27e760","type":"simple","updated_at":"2022-02-10T23:29:56.454Z","updated_by":"elastic"}
```

### Exception item after workaround applied
```ndjson
{"_version":"WzEzNzQwLDFd","comments":[{"comment":"I'm a comment"}],"created_at":"2022-02-09T20:11:39.280Z","created_by":"elastic","description":"Rule w exceptions - exception list item","entries":[{"field":"_index","type":"match","value":",nkjh","operator":"included"}],"id":"7d813010-89e4-11ec-9a39-157f181d84ab","item_id":"a09d05c7-495c-40ff-b8a8-3ceafd184951","list_id":"simple_list","name":"Rule w exceptions - exception list item","namespace_type":"single","os_types":[],"tags":[],"tie_breaker_id":"014c8b06-1ce0-4f6d-a97c-88795f27e760","type":"simple","updated_at":"2022-02-10T23:29:56.454Z","updated_by":"elastic"}
```
